.gitignore

```diff
@@ -34,5 +34,6 @@ node_modules
 
 #local env file
 .env.local
```

Comment:
I think this line can be deleted because README changed, and this file is not created anymore